### DISCUSSION TITLE
[question] 请问，是否在使用G6上，不管是使用Vue还是React框架都没有差异性是吗？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#6467] [Q&A] [answered]
G6是基于js的可视化框架：
> A Graph Visualization Framework in JavaScript.

请问，是否在使用G6上，不管是使用Vue还是React框架都没有差异性是吗？也就是说：假如在Vue上如果有bug，那么在React上也一样有bug。

**Comment by Rey-Wang** [comment, marked as answer]
取决于什么 bug，graph 的初始化和框架渲染的生命周期相关，涉及这块的问题肯定和框架相关的，如果是 canvas 的渲染问题，应该没有差异
